src/index.html

```diff
@@ -17,20 +17,18 @@
   </head>
   <body class="content">
     <section class="card" data-qa="card">
-      <a href="/" class="card__img-link">
-        <img
-          class="card__img"
-          src="images/imac.jpeg"
-          alt="iMac" >
-      </a>
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="iMac" >
```

Comment:
<img
        class="card__img"
        src="images/imac.jpeg"
        alt="iMac"
>

better like this, check it everywhere (51-54)

Reply:
thanks